usage.md

```diff
@@ -6,31 +6,47 @@
 // address: Smart contract address of app
 // provider: Web3provider (Eg: If you have installed metamask then window.ethereum will work)
 // Default value of provider is window.ethereum
-const arcanaInstance = new arcana.storage.StorageProvider({ appId, provider, email });
+import { StorageProvider } from '@arcana/storage';
+
+const storage = new StorageProvider({ appId, provider, email });
 ```
 
-## Get Uploader
+## Upload a File object
 
-```typescript
-const Uploader = arcanaInstance.getUploader();
-// file: Blob format
+The `file` object must be an instance of a `Blob` or a descendant (`File`, etc.).
+
+#### v0.3 (for simpler use cases)
+```javascript
+storage.upload(file, (bytesUploaded, bytesTotal) => { console.log('Progress:', (bytesUploaded / bytesTotal).toFixed(2), '%')});
```

Comment:
Can we expand this code snippet to include handling success and error?

Reply:
Done.